wiki/Bazel-Setup-Instructions-for-Linux.md

```diff
@@ -1,31 +1,29 @@
 ## Instructions
 
 **The steps to install Bazel on Linux are:**
-1. Install Bazel 
-2. Install OpenJDK 8
+1. Install Bazel
+2. Install OpenJDK 11
 3. Install Python 2 and make sure it is active in your environment
 4. Set up the ANDROID_HOME environment variable
 5. Prepare the build environment
 6. Verify the Android build
 
 ### 1. Install Bazel
 
-Install Bazel from [here](https://docs.bazel.build/versions/master/install.html). Make sure that you follow the instructions for installing a specific version (Oppia Android requires 4.0.0 and won't build on other versions).
+Install Bazel from [here](https://docs.bazel.build/versions/master/install.html). Make sure that you follow the instructions for installing a specific version (Oppia Android requires 6.5.0 and won't build on other versions).
    - Note: if you find any errors related to `cURL`, please set up cURL on your machine. For Linux, you can use `sudo apt install curl`.
 
-### 2. Install OpenJDK 8
+### 2. Install OpenJDK 11
```

Comment:
Would it be possible to add $JAVA_HOME setup instructions in this PR? Re: https://github.com/oppia/oppia-android/issues/5293

Reply:
I think we actually don't need this anymore with the other latest changes. Now that we're using the JDK bundled with Bazel, developers technically don't even need to install Java anymore (as far as I can tell). I've cleaned up these docs significantly in the latest changs.